CONTRIBUTING.md

```diff
@@ -6,6 +6,10 @@ When opening new issues or commenting on existing issues on this repository
 please make sure discussions are related to concrete technical issues with the
 Node.js software.
 
+For technical issues related to using Node.js that are not issues with the
```

Comment:
What do you think of something more like this: 'For general help using Node.js, please file an issue at...`   I think that may be easier to grok than "issues related to using Node.js that are not issues with the Node.js software itself"...